fdbclient/HTTP.actor.cpp

```diff
@@ -334,7 +334,7 @@ namespace HTTP {
 			}
 
 			// Write headers to a packet buffer chain
-			PacketBuffer *pFirst = new PacketBuffer();
+		    PacketBuffer* pFirst = PacketBuffer::create();
```

Comment:
Nit: weird whitespace

Reply:
We need to rollback this entire file anyways - you reformatted everything which makes a review hard